requirements-optional.txt

```diff
@@ -1 +1,3 @@
-skater==1.1.2
\ No newline at end of file
+skater==1.1.2
+Keras==2.2.4
```

Comment:
I think it's ok to actually add keras/tensorflow requirements to the requirements.txt, not the optional reqs. I mostly put skater in here because its build is failing on its own repo and its not pip-installable (wordcloud requirement build fails)